content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Notation v1.0.0-RC.6 is available!
+author:  "Notation Release Team"
+date: 2023-05-26
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of [Notation CLI](https://github.com/notaryproject/notation) v1.0.0-rc.6, [notation-go library](https://github.com/notaryproject/notation-go) v1.0.0-rc.6, and [notation-core-go library](https://github.com/notaryproject/notation-go) v1.0.0-rc.4. This blog walks you through the updates in this release.
+
+## What's new
+
+This release adds the following major changes:
+
+- TODO Add security CVEs
+- Notation commands support reading docker credentials if credentials store is not installed
```

Comment:
```suggestion
- Notation commands support reading Docker credentials if the credentials store is not configured
```

Reply:
See https://docs.docker.com/engine/reference/commandline/login/#configure-the-credentials-store